src/database/models/vendor.js

```diff
@@ -13,27 +16,59 @@ export default (sequelize, DataTypes) => {
     createdAt: {
       allowNull: false,
       type: DataTypes.DATE,
+      defaultValue: DataTypes.NOW,
       field: 'created_at',
     },
     updatedAt: {
       allowNull: false,
       type: DataTypes.DATE,
+      defaultValue: DataTypes.NOW,
       field: 'updated_at',
     },
   }, {});
 
+  Vendor.beforeCreate(async vendor => {
+    vendor.password = await vendor.generatePasswordHash();
+  });
+
+  Vendor.prototype.generatePasswordHash = async function generatePasswordHash() {
+    const saltRounds = +process.env.SALT;
```

Comment:
Why is there + in front of process.env.SALT

Reply:
Our `process.env.SALT` value is a number. We are putting the `+` sign to cast it to an integer